src/main/resources/assets/origins/lang/es_es.json

```
@@ -1,170 +1,243 @@
+  
+  "power.origins.lay_eggs.name": "Ovíparo",
+  "power.origins.lay_eggs.description": "Todas las mañanas cuando despiertes, pondrás un huevo.",
+  
+  "power.origins.phantomize.name": "Forma Fantasma",
+  "power.origins.phantomize.description": "Puedes cambiar entre forma fantasma y humana a voluntad, pero si te agotas lo suficiente para no poder correr, vuelves a tu forma humana.",
+  
+  "power.origins.like_air.name": "Como el Viento",
+  "power.origins.like_air.description": "Tu velocidad en la tierra se mantiene igual en el aire.",
+  
+  "power.origins.velvet_paws.name": "Patas Acolchadas",
+  "power.origins.velvet_paws.description": "Las almohadillas en tus patas hacen que tus pisadas no causen vibraciones, lo que hace que no te puedan detectar los Sensores de Sculk.",
+  
+  "category.origins": "Orígenes",
+  "key.origins.primary_active": "Poder Activo (Principal)",
+  "key.origins.secondary_active": "Poder Activo (Secundario)",
+  "key.origins.view_origin": "Ver Origen",
+  
+  "text.autoconfig.origins.title":   "origins Configuration",
+  "text.autoconfig.origins.option.xOffset": "Cooldown Bar Offset X",
+  "text.autoconfig.origins.option.yOffset": "Cooldown Bar Offset Y",
+  "text.autoconfig.origins.option.phantomizedOverlayStrength": "Phantomized Overlay Strength",
+  
+  "item.origins.orb_of_origin": "Orbe de Orígenes",
+  "item.origins.orb_of_origin.layer_generic": "Te deja escoger un nuevo %s.",
+  "item.origins.orb_of_origin.layer_specific": "Asigna tu %1$s a \"%2$s\".",
+  
+  "enchantment.origins.water_protection": "Protección Acuática",
+  "enchantment.origins.water_protection.desc": "Retrasa el daño por hidrofobia.",
+  
+  "container.shulker_inventory_power": "Inventario de Shulker",
+  
+  "origins.gui.select": "Seleccionar",
+  "origins.gui.close": "Cerrar",
+  "origins.gui.choose": "Elige",
+  "origins.gui.choose_origin.title": "Elige tu %s.",
+  "origins.gui.view_origin.title": "Este es tu %s.",
+  "origins.gui.view_origin.empty": "No puedes tener ningún origen.",
+  "origins.gui.view_origin.not_installed": "El servidor no tiene Origins instalado.",
+  "origins.gui.impact.impact": "Impacto",
+  "origins.gui.impact.none": "Ninguno",
+  "origins.gui.impact.low": "Bajo",
+  "origins.gui.impact.medium": "Intermedio",
+  "origins.gui.impact.high": "Alto",
+  "origins.gui.version_mismatch": "Este servidor está usando una versión de Origins (v%1$s) que es incompatible con la que tienes instalada (v%2$s)",
+  
+  "origins.gui.badge.active": "Habilidad activa. Presione %s para activarla.",
+  "origins.gui.badge.toggle": "Alternar con %s.",
+  "origins.gui.badge.recipe.crafting.shaped": "Receta de elaboración (con forma) proporcionada por el poder:",
+  "origins.gui.badge.recipe.crafting.shapeless": "Receta de elaboración (sin forma) proporcionada por el poder:"
```

You missed a comma here.
```suggestion
  "origins.gui.badge.recipe.crafting.shapeless": "Receta de elaboración (sin forma) proporcionada por el poder:",
```